examples/application_commands/autocompleted_command.py

```diff
@@ -0,0 +1,33 @@
+import nextcord
+from nextcord import Interaction, SlashOption
+
+client = nextcord.Client()
+
+list_of_dog_breeds = [  # the list of dog breeds
+    "German Shepard",
+    "Poodle",
+    "Pug",
+    "Shiba Inu",
+]
+
+
+@client.slash_command(guild_ids=[...])  # Limits guilds
+async def your_favorite_dog(
+    interaction: Interaction,
+    dog: str = SlashOption(
+        name="dog",  # the name
+        description="Choose the best dog from this autocompleted list!",  # our description
+    ),
+):  # our slash option.
+    await interaction.response.send_message(
+        f"your favorite dog is {dog}!"
+    )  # sends the autocompleted result
+
+
+@your_favorite_dog.on_autocomplete("dog")
+async def favorite_dog(interaction: Interaction, dog):
+
+    await interaction.response.send_autocomplete(list_of_dog_breeds)
```

Comment:
Why is this an autocomplete example and yet sending a static/constant list of items? This should send a filtered list of items matching what the user has typed, as per the idea of autocomplete options.